packages/core/src/project-structure.ts

```diff
@@ -0,0 +1,19 @@
+import findUp from 'find-up';
+
+const JS_CONFIG_FILENAME = 'kuai.config.js';
+const TS_CONFIG_FILENAME = 'kuai.config.ts';
+
+export function isCwdInsideProject(): boolean {
+  return findUp.sync(TS_CONFIG_FILENAME) !== null || findUp.sync(JS_CONFIG_FILENAME) !== null;
+}
+
+export function getUserConfigPath(): string | undefined {
+  const tsConfigPath = findUp.sync(TS_CONFIG_FILENAME);
+  if (tsConfigPath) {
+    return tsConfigPath;
+  }
+
+  const pathToConfigFile = findUp.sync(JS_CONFIG_FILENAME);
+
+  return pathToConfigFile;
```

Comment:
Exception of `not found` is not handled

Reply:
Here if not found will return undefined, in the initialization of `KuaiContext` will not force must have a `kuai.config.ts`, do we want to force the user to use when there must be a config file, if not on the error?